python/tank/api.py

```diff
@@ -592,10 +594,38 @@ def preview_filesystem_structure(self, entity_type, entity_id, engine=None):
                                                       engine)
         return folders
 
+    def get_site_cache_root(self):
+        """
+        Returns the location of the site cache root for this Tank instance.
+        """
+        # Invoke the global method which is generic and can compute any site
+        # cache root.
+        global get_site_cache_root
+        return get_site_cache_root(self.shotgun.base_url)
 
 ##########################################################################################
 # module methods
 
+
+def get_site_cache_root(base_url):
```

Comment:
I added the method. I wasn't sure it having tank.api import stuff from util would be fine. If it is, by all means let's do it.

I think it would help if we ran a static analysis tool on tank to know what's the dependency tree.